cassandra/cluster.py

```diff
@@ -2189,7 +2189,7 @@ def shutdown(self):
             else:
                 self.is_shutdown = True
 
-        for pool in self._pools.values():
+        for pool in list(self._pools.values()):
```

Comment:
I checked the source of six.itervalues and I think it would not have fixed the issue, so your is all good. Thanks